src/Core/Checkout/Customer/SalesChannel/AddWishlistProductRoute.php

```diff
@@ -61,21 +61,31 @@ public function getDecorated(): AbstractAddWishlistProductRoute
      * @Since("6.3.4.0")
      * @OA\Post(
      *      path="/customer/wishlist/add/{productId}",
-     *      summary="Add a product into customer wishlist",
+     *      summary="Add a product to a wishlist",
+     *      description="Adds a product to a customers wishlist.
+
+     **Important constraints**
```

Comment:
Indentation has to be 0 - see screenshot.

![image](https://user-images.githubusercontent.com/7780750/117157103-33d23780-adb6-11eb-8015-c1d7f8ff7f05.png)